docs/notebook.jl

```diff
@@ -0,0 +1,160 @@
+md"It is generally best to convert this table to a `DataFrame` for display or summary"
+
+# ╔═╡ 9f327062-76e3-11eb-16dc-7be3a2b1c09f
+df = DataFrame(frm)
+
+# ╔═╡ b0202450-76e3-11eb-3218-8bddeed1c50d
+describe(df)
+
+# ╔═╡ 0fce98d0-76e4-11eb-0ed4-430ae7972fce
+md"Note that if you go back to the entry field of `sub_n` or `item_n` and change the value, the other results are automatically updated"
+
+# ╔═╡ 34b8677c-76e4-11eb-1807-9306323a7c9f
+md"""
+## Fitting a model to the inert data
+
+In the `MixedModels` package the formula language is similar to that in the `lme4` package for `R` with the exception that it must be wrapped in a call to the macro `@formula`
+"""
+
+# ╔═╡ 48e92618-76d1-11eb-053c-e77449325b9c
+m0form = @formula dv ~ 1 + cond + (1|subj) + (1|item);
+
+# ╔═╡ 7292d5fa-76e4-11eb-243f-8f7ad404d683
+md"Also, contrasts are specified in the call to fit the model instead of being part of the metadata for the factor"
+
+# ╔═╡ 761fb192-76d1-11eb-2f8e-890af7ae08d8
+m0 = fit(MixedModel, m0form, frm, contrasts=Dict(:cond => EffectsCoding()))
+
+# ╔═╡ 91169880-76d1-11eb-28da-0f4135f0e995
+md"""
+As this dependent variable is inert (i.e. simulated from a standard Normal distribution without any fixed- or random-effects contributions) it is not surprising the the parameter estimates for the variance components for subject and item are close to zero as are the estimates for the fixed-effects.
+
+## Simulated model fits for non-inert data
+
+To simulate responses from non-intert data we must specify values for the parameters in the form of a scalar, `σ`, the standard deviation of the per-observation noise, and two vectors: `β` the fixed-effects parameter vector and `θ` the vector of relative standard deviations.
+
+The fixed-effects parameter is relative to the model matrix, `X`, which, in this case, is the overall mean DV for a typical condition and half the difference between the `hard` and `easy` conditions.  It is half the difference because we use a +/- 1 coding for the `cond` factor, producing a model matrix, `X`, of
+"""
+
+# ╔═╡ d217cc16-76e8-11eb-2f64-6ff4931c87e9
+Int.(m0.X')   # convert to integers to save space in the display
+
+# ╔═╡ 9d46c892-76e9-11eb-008e-67d5100eb3fa
+md"Suppose we want an overall mean of 400 ms. and the difference between `hard` and `easy` to be 50 ms., then we would set"
+
+# ╔═╡ dbd15424-76e9-11eb-2306-436929c4e756
+β = [400.0, 25.0]   # Note we include the decimal point to force Float64 values
+
+# ╔═╡ fbdfaf02-76e9-11eb-2404-d9a288cf1d31
+md"We also set"
+
+# ╔═╡ 167234ae-76ea-11eb-2589-93a3e6940453
+begin
+	σ  =  25.0  # s.d. of per-observation noise
+	σ₁ = 100.0  # s.d. of random effects for subject
+	σ₂ =  50.0  # s.d. of random effects for item
+	θ = [σ₁, σ₂] ./ σ
+end;
+
+# ╔═╡ b75c8580-76eb-11eb-2cca-37a0702e44a5
+md"And now simulate a few thousand cases."
+
+# ╔═╡ cd924f92-76eb-11eb-3275-eb4f55282d5d
+sim = parametricbootstrap(MersenneTwister(12321), 5000, m0; β=β, σ=σ, θ=θ);
```

Comment:
Would it make sense to look at `DataFrame(first(sim.coefpvalues))` to show what the (fixed of effects of the) mixed model looks like?

Reply:
Sure.  I was just exploring somewhat here so my choice of what to show and what not to show was somewhat arbitrary.